.github/workflows/scheduled_integration_test.yaml

```diff
@@ -13,7 +13,7 @@ jobs:
       extra-arguments: -x --localstack-address 172.17.0.1
       pre-run-script: localstack-installation.sh
       trivy-image-config: "trivy.yaml"
-      juju-channel: 3.4/stable
+      juju-channel: 3.6/stable
```

Comment:
I splitted the workflow in 2 to prevent running integration tests if the PR only changes the terraform but now I see that this could be done in only 1 workflow by adding a "if" to the "check-changes". wdyt